### ISSUE TITLE
InvalidOperationException : Constructor not found for type

### ISSUE THREAD

**Issue by alexey-butalov-aspose** [microsoft/node-api-dotnet#232]
The error occurs for the sample project:
```
CSC : error NAPI1001: InvalidOperationException : Constructor not found for type: class2   
at Microsoft.JavaScript.NodeAp i.Generator.SymbolExtensions.AsConstructorInfo(IMethodSymbol methodSymbol)   
at System.Linq.Enumerable.WhereSelectEnumera bleIterator`2.ToArray()   
at Microsoft.JavaScript.NodeApi.Generator.ModuleGenerator.ExportType(SourceBuilder& s, ITypeSym bol type, String exportName)   
at Microsoft.JavaScript.NodeApi.Generator.ModuleGenerator.ExportModule(SourceBuilder& s, I TypeSymbol moduleType, IEnumerable`1 exportItems)   
at Microsoft.JavaScript.NodeApi.Generator.ModuleGenerator.GenerateMod uleInitializer(ISymbol moduleInitializer, IEnumerable`1 exportItems)   
at Microsoft.JavaScript.NodeApi.Generator.ModuleGe nerator.Execute(GeneratorExecutionContext context)
[ConsoleApp1.csproj] 
```
[ConsoleApp1.zip](https://github.com/microsoft/node-api-dotnet/files/14606776/ConsoleApp1.zip)
Command:
```
dotnet publish -c DEBUG -r win-x64
```
0.5.12 nuget version

**Comment by jasongin** [comment]
Thanks for the bug report with sample project code. I just submitted a small PR to fix it: #233

**Comment by alexey-butalov-aspose** [comment]
Thanks. The issue is fixed.
